plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaTextDocumentServiceWraper.java

```diff
@@ -10,13 +10,40 @@
  */
 package org.eclipse.che.plugin.java.languageserver;
 
+import static org.eclipse.che.api.fs.server.WsPathUtils.SEPARATOR;
+import static org.eclipse.che.api.languageserver.service.LanguageServiceUtils.extractProjectPath;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.List;
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import org.eclipse.che.api.fs.server.WsPathUtils;
+import org.eclipse.che.api.languageserver.exception.LanguageServerException;
 import org.eclipse.lsp4j.CodeActionParams;
 import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.DocumentFormattingParams;
+import org.eclipse.lsp4j.FormattingOptions;
+import org.eclipse.lsp4j.TextEdit;
 import org.eclipse.lsp4j.services.TextDocumentService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
 
 public class JavaTextDocumentServiceWraper {
+  private static final Logger LOG = LoggerFactory.getLogger(JavaTextDocumentServiceWraper.class);
+
+  private static final String CHE_FOLDER = ".che";
+  private static final String CHE_FORMATTER_XML = "che-formatter.xml";
+  private static final String PROJECTS_ROOT = "/projects";
```

Comment:
It is possible to use `che.workspace.projects.storage=/projects` property?

Reply:
I can't use injection here. Will use this constant from `LanguageServiceUtils`